### PR TITLE
#717 Changed the request view buttons to use NER buttons

### LINKED ISSUE #717: [Frontend] - Change ReviewChangeRequestView to use NER Buttons
### Description

In the pursuit of making the looking and feel of our app consistent, we wanna use new components that have been made (NERSuccessButton/NERFailButton) in every place where we currently have a cancel/save button combo. By doing this, we not only make all these save/cancel buttons adhere to the new look, but we also allow ourselves to change how all these save buttons look in the future by changing only one file!

For this ticket, change the save/cancel buttons in the ReviewChangeRequestView component to use the NERSuccessButton/NERFailButton component rather than just the plain MUI button.

Put the cancel on the left, and the submit on the right

### Acceptance Criteria

The save/cancel buttons on ReviewChangeRequestView use NERSuccessButton/NERFailButton and not Button
The save/cancel buttons on that page looks like the new success/cancel buttons (green and red contained, see the edit members section on a teams specific page to see a good example of how it looks).
Verify nothing looks weird/ everything works as it did before.
Buttons should be 2 px apart, good example: https://github.com/Northeastern-Electric-Racing/FinishLine/pull/731

### Proposed Solution

All the props should be the same, all you need to do is swap out the old for the new / for the Save/Cancel buttons respectively.

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by Aaryan1203**
## Changes

Changed the request view buttons to use NERsuccess/fail buttons

Screenshot 2023-02-02 at 2 22 32 PM

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #717

### CODE REVIEW

**Review by Peyton-McKee — DISMISSED** [review]
Lgtm!

File: src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx [hunk 1]
```diff
@@ -205,26 +207,24 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
...
             onClick={() => handleAcceptDeny(false)}
           >
             Deny
-          </Button>
+          </NERFailButton>
+          <NERSuccessButton
+            variant="contained"
+            type="submit"
+            form="review-notes-form"
+            sx={{ mx: 1, textTransform: 'none', fontSize: 16 }}
```
Comment (RChandler234) [inline src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx:223]: I don't think you need textTransform or fontsize, none of the other fail/success buttons are using these. See if it still looks good/standard with the buttons on the new change request form on develop if you remove those two on both buttons.

**Review by RChandler234 — CHANGES_REQUESTED** [review]

File: src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx (hunk 1 above)
Comment (Aaryan1203) [reply src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx:223]: Ok will do, do I edit this on the same branch or make a new branch for it?

Comment (RChandler234) [reply src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx:223]: edit it on the same branch, add and commit the changes, then if you git push again, it'll automatically push the changes to this pull request

**Review by Peyton-McKee — APPROVED** [review]
LGTM

**Comment by RChandler234** [comment]
@Aaryan1203 just now seeing these because I loaded up the app, but you need to change the other set of accept/deny buttons in this file as well. We have two because there are two versions of the review change request view, one with proposed solutions and one without.

**Review by RChandler234 — CHANGES_REQUESTED** [review]
just now seeing these because I loaded up the app, but you need to change the other set of accept/deny buttons in this file as well. We have two because there are two versions of the review change request view, one with proposed solutions and one without. The set you already did looks perfect though, should it should hopefully be quick

File: src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx [hunk 2]
```diff
@@ -22,6 +22,8 @@ import {
   Breakpoint
 } from '@mui/material';
```
Comment (Peyton-McKee) [inline src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx:23]: Linting error make sure to remove any components you dont end up using

File: src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx [hunk 3]
```diff
@@ -148,29 +150,27 @@ const ReviewChangeRequestsView: React.FC<ReviewChangeRequestViewProps> = ({
           </form>
         </DialogContent>
         <DialogActions>
-          <Button
-            color="success"
+          <NERFailButton
+            className={'ml-3'}
             variant="contained"
```
Comment (Peyton-McKee) [inline src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx:155]: I dont know if we need this cant tell the difference when Its deleted

**Review by Peyton-McKee — CHANGES_REQUESTED** [review]

File: src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx (hunk 3 above)
Comment (Aaryan1203) [reply src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx:145]: Which part do we not need?

Comment (Peyton-McKee) [reply src/frontend/src/pages/ChangeRequestDetailPage/ReviewChangeRequestView.tsx:145]: The classname

**Review by Peyton-McKee — APPROVED** [review]
This looks good

**Pull request merged**